README.md

```diff
@@ -194,36 +193,16 @@ Pre-requisites established by that script:
 * `kind` executable installed at `/usr/local/bin/kind`
 * Kind cluster named `kcp`
 * Cert-manager installer and running
```

Comment:
Please reword to `cert-manager installed and running` (note especially the lowercase c)